src/mumble/ALSAAudio.cpp

```diff
@@ -209,16 +209,26 @@ ALSAEnumerator::ALSAEnumerator() {
 	snd_card_next(&card);
 	while (card != -1) {
 		char *name;
+		int err;
 		snd_ctl_t *ctl = nullptr;
-		snd_card_get_longname(card, &name);
+		err = snd_card_get_longname(card, &name);
+		if (err != 0) {
+			Global::get().mw->msgBox(
+				tr("Getting name of the sound card failed: %1").arg(QString::fromLatin1(snd_strerror(err)).toHtmlEscaped()));
+			return;
+		}
```

Comment:
```suggestion
		if ((err = snd_card_get_longname(card, &name)) != 0) {
			Global::get().mw->msgBox(
				tr("Getting name (longname) of the sound card failed: %1").arg(QString::fromUtf8(snd_strerror(err)).toHtmlEscaped()));
			return;
		}
```

Reply:
> Putting the assignment of err into the if-statement is simply a slightly more condensed way of getting this done.

OK, got it!

> I know that other parts in the code use fromLatin1 for parsing the returned error string, but given that this is an external String, we should default to fromUtf8 instead, just in case that's needed :)

Oh, you got me. LOL. I don't know the right way to print the error message so just copy it from other parts of the code.